client/src/ConfiguratorPage/ConfiguratorPage.js

```diff
@@ -40,15 +40,17 @@ export const ConfiguratorPage = () => {
     },
   });
 
-  if (isLoading) {
+  if (status === "loading") {
```

Comment:
try to always use constants for things like that.

something like status === IngredientStatus.LOADING

in typescript you would make IngredientStatus — enum. 

in plain javascript, you can do something like:

const IngredientStatus = {
   LOADING: 'loading'
}